.circleci/config.yml

```diff
@@ -5,35 +5,33 @@ orbs:
 jobs:
   build:
     docker:
-      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester-v4
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
     steps:
       - checkout
       - run:
           name: 'Setup virtual env'
           command: |
-            virtualenv -p python3 ~/.virtualenvs/tap-google-sheets
-            source ~/.virtualenvs/tap-google-sheets/bin/activate
+            python3 -m venv /usr/local/share/virtualenvs/tap-google-sheets
```

Comment:
are you sure this directory will always exist?  The home directory ~ will.

Reply:
This path is an environment variable in tap-tester now that we are using the newer image.